aio/local_packages_util.bzl

```
@@ -32,6 +34,8 @@ def substitute_local_packages(deps):
     """
     substituted = []
     for dep in deps:
+        if dep.startswith(":"):
+            continue
```

Could we switch the direction of the check here?

```suggestion
        if dep.startswith("@aio_npm//"):
        # actually do the stuff
```